variables.tf

```diff
@@ -12,6 +12,30 @@ variable "name" {
   description = "The name to give the IBM Event Notification instance created by this module."
 }
 
+variable "destination_name" {
+  type        = string
+  description = "The Destintion name."
+  default     = null
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "The bucket name in IBM cloud object storage instance."
+  default     = null
+}
+
+variable "cos_instance_id" {
+  type        = string
+  description = "The instance id for IBM Cloud object storage instance."
```

Comment:
The ID of the IBM Cloud Object Storage instance in which the bucket defined in the `cos_bucket_name` variable exists